lib/ansible/module_utils/basic.py

```diff
@@ -516,6 +516,18 @@ def is_executable(path):
             or stat.S_IXOTH & os.stat(path)[stat.ST_MODE])
 
 
+class AnsibleFallbackNotFound(Exception):
+    pass
+
+def env_fallback(*args):
```

Comment:
Should add **kwargs (that aren't used by this fallback method but could be used by others) as well.